style.css

```diff
@@ -1 +1,268 @@
 /*  Add your styling here */
+body{
```

Comment:
You have 2 files with the same name `style.css`? If you want to split your CSS, make sure you use different file names. If you only need one, then delete the other one 🙂

Reply:
Deleted the redundant one :)